SillySCP/Commands/CancelVolunteer.cs

```diff
@@ -0,0 +1,49 @@
+﻿using CommandSystem;
+using Exiled.API.Extensions;
+using Exiled.API.Features;
+using PlayerRoles;
+using SillySCP.API.Features;
+
+namespace SillySCP.Commands
+{
+    [CommandHandler(typeof(RemoteAdminCommandHandler))]
+    public class CancelVolunteer : ICommand
+    {
+        public string Command { get; } = "cancelvolunteer";
+        public string Description { get; } = "Cancel a volunteer prompt";
+        public string[] Aliases { get; } = new[] { "cv", "rv", "rvol", "cvol","revokevolunteer"};
+
+        public bool Execute(
+            ArraySegment<string> arguments,
+            ICommandSender sender,
+            out string response
+        )
+
+        {
+            if (arguments.Count != 1)
+            {
+                response = "Usage: cancelvolunteer <SCP>";
+                return false;
+            }
+
+            if (!VolunteerSystem.VaildScps.TryGetValue(arguments.At(0), out RoleTypeId role))
+            {
+                response = "Must be a SCP";
+                return false;
+            }
+
+            response = "Canceling " + role.GetFullName();
+            if (VolunteerSystem.Volunteers.Count < 1)
+            {
+                response = "You do not have any volunteers yet";
+                return false;
+            }
+
+            var volunteer = VolunteerSystem.Volunteers.Find(v => v.Replacement == role);
```

Comment:
no var, and when you do TryGetFirst, remove this, as it will give you the volunteer instance